CHANGELOG.txt

```diff
@@ -7,8 +7,8 @@ Changelog
  * Maintenance: Update BeautifulSoup upper bound to 4.12.x (scott-8)
 
 
-5.2 (xx.xx.xxxx) - IN DEVELOPMENT
-~~~~~~~~~~~~~~~~
+5.2 LTS (1.11.2023)
```

Comment:
Please don't fill in the release date or remove the "in development" text yet - we'll do that when the release happens.

Reply:
Oh. I will correct that. Thanks

Reply:
@gasman I filled in the release date in the 5.2.md file (based on the 5.2 milestone). Should I change that back to unreleased?

Reply:
Yes, please change that back to unreleased.